ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot

```diff
@@ -1,17 +1,17 @@
 *** Settings ***
-Documentation       Test suite for OpenShift Pipeline
+Documentation       Test suite for OpenShift Pipeline API
 Resource            ../../../Resources/RHOSi.resource
 Resource            ../../../Resources/ODS.robot
 Resource            ../../../Resources/Common.robot
 Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/Pipelines/OpenShiftPipelines.robot
 Library             DateTime
 Library             ../../../../libs/DataSciencePipelinesAPI.py
 Suite Setup         Data Science Pipelines Suite Setup
 Suite Teardown      RHOSi Teardown
 
 
 *** Variables ***
-${REDHAT_OPENSHIFT_PIPELINES_YAML}=            ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
 ${DATA_SCIENCE_PIPELINES_APPLICATION_YAML}=    ods_ci/tests/Resources/Files/data-science-pipelines-sample.yaml
```

Comment:
should this be moved to the Resource file like done with line 14?

Reply:
it would also avoid to define this variable twice (here and in ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/432__data-science-pipelines-tekton.robot)

Reply:
Done